libs/common/src/types/mongo-db-models/index.ts

```diff
@@ -7,4 +7,4 @@ export * from './users';
 export * from './deletion-audit-logs';
 export * from './tfm-facilities';
 export * from './facility';
-export * from './tfm-deals';
+export * from './tfm-deal';
```

Comment:
It looks like the other files here use the plural (as in the same as the collection name)

Reply:
Changed to the plural👍